com.gitlab.jakobdev.jdnbtexplorer.yml

```
@@ -0,0 +1,25 @@
+app-id: com.gitlab.jakobdev.jdnbtexplorer
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: jdNBTExplorer
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --share=ipc
  - --device=dri
  - --socket=wayland
```

IPC is needed for X11 and DRI is needed for openGL (which I suspect is used here)